docs/how-to/drbd-host-networking.md

```diff
@@ -0,0 +1,100 @@
+      patch: |
+        apiVersion: v1
+        kind: Pod
+        metadata:
+          name: satellite
+        spec:
+          hostNetwork: true
+```
+
+After the Satellite Pods are recreated, they will use the host network. Any existing DRBD resources are reconfigured
+to use the new IP Address instead.
+
+## Switch from Host Network to Container Network
+
+Switching back from host network to container network involves manually resetting the configured peer addresses used by
+DRBD. This can either be achieved by rebooting every node, or by manually resetting the address using `drbdadm`.
+
+### Switch from Host Network to Container Network Using Reboots
+
+First, you need to remove the `LinstorSatelliteConfiguration` that set `hostNetwork: true`:
+
+```
+$ kubectl delete linstorsatelliteconfigurations.piraeus.io host-network
+linstorsatelliteconfiguration.piraeus.io "host-network" deleted
+```
+
+Then, reboot each cluster node, either one by one or multiple at once. In general, replication will not work between
+rebooted nodes and non-rebooted nodes. The non-rebooted nodes will continue to use the host network addresses, which
+are generally not reachable from the container network.
+
+After all nodes are rebooted, all resources are configured to use the container network, and all DRBD connections
+should be connected again.
+
+### Switch from Host Network to Container Network Using `drbdadm`
+
+During this procedure, ensure no new volumes or snapshots are created: otherwise the migration to the
+container network might not be applied to all resource.
+
+First, you need to temporarily stop all replication and suspend all DRBD volumes using `drbdadm suspend-io all`.
+The command is executed once on each LINSTOR Satellite Pod.
+
+```
+$ kubectl exec node1.example.com -- drbdadm suspend-io all
+$ kubectl exec node2.example.com -- drbdadm suspend-io all
+$ kubectl exec node3.example.com -- drbdadm suspend-io all
+```
+
+Next, you will need to disconnect all DRBD connections on all nodes.
+
+```
+$ kubectl exec node1.example.com -- drbdadm disconnect --force all
+$ kubectl exec node2.example.com -- drbdadm disconnect --force all
+$ kubectl exec node3.example.com -- drbdadm disconnect --force all
+```
+
+Now, we can safely reset all DRBD connection paths, which frees the connection to be moved to the container network.
+
+```
+$ kubectl exec node1.example.com -- drbdadm del-path all
+$ kubectl exec node2.example.com -- drbdadm del-path all
+$ kubectl exec node3.example.com -- drbdadm del-path all
+```
+
+Finally, removing the `LinstorSatelliteConfiguration` that set `hostNetwork: true` will trigger the creation of new
+LINSTOR Satellite Pods using the container network:
+
+```
+$ kubectl delete linstorsatelliteconfigurations.piraeus.io host-network
+linstorsatelliteconfiguration.piraeus.io "host-network" deleted
+```
+
+After the Pods are recreated and the LINSTOR Satellites are `Online`, the DRBD resource will be reconfigured and resume
```

Comment:
Interesting. I would have expected an explicit `resume-io` to be necessary. Does LINSTOR do that automatically nowadays?

Reply:
Seems that way, I quickly tested the procedure, and after the LINSTOR Satellite is online the DRBD resources are no longer suspended